src/DynamoCoreWpf/ViewModels/PackageManager/PackageManagerClientViewModel.cs

```diff
@@ -900,6 +905,21 @@ internal async void ExecutePackageDownload(string name, PackageVersion package,
                     }
                 }
 
+                //if any of the required packages use a pre 3.x version of Dynamo, show a dialog to the user
+                //allowing them to cancel the package download
+                if (preDYN3Deps.Any())
+                {
+                    var res = MessageBoxService.Show(ViewModelOwner,
+                    string.Format(Resources.MessagePackageOlderDynamo, DynamoViewModel.BrandingResourceProvider.ProductName),
+                    string.Format(Resources.PackageUseOlderDynamoMessageBoxTitle, DynamoViewModel.BrandingResourceProvider.ProductName),
```

Comment:
Do we want to limit the message just to say the package or one of its dependencies uses an older version of _Dynamo_ than what is currently being used OR do we want the message to explicitly say that the package is a pre 3.0 package, and that the current version of Dynamo is >= 3.0?

Reply:
I would prefer that the warning be a simple guidance similar to what exists on the package manager. "Older version of Dynamo, and may behave in unexpected ways".

Saying that it is a pre-3.0 package or other similar (possibly negative connotation) languages could send people into a frenzy. This happens quite a bit with the warning "Package Name loaded a version of RevitAPI.dll that is different". People get pretty worked up and blame the package authors.

Reply:
could we add a link to some dynamobim.org developer documentation on this topic. That way we could keep the message less alarming and point to some other place for more info

Reply:
Primer link makes sense to me too.

Reply:
Developer Primer makes perfect sense to me. Then we only need to keep the hyper-link truthful and can update at will.